### PR TITLE
Revert "Improve usability of Select component"

### PR DESCRIPTION
**Pull request by dinahshi**
This reverts commit bd24484a2c1888106ed5161e517c5f7f0b8eae1c.

@igillis @thangdinh

### CODE REVIEW

File: src/select.coffee [hunk 1]
```diff
@@ -64,10 +64,10 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
   placeholder:        undefined
   disabled: no
 
+  # we need to set tabindex so that div responds to key events
```
Comment (igillis) [inline src/select.coffee]: missed comment

**Comment by igillis** [comment]
lgtmbc

File: src/select.coffee (hunk 1 above)
Comment (dinahshi) [inline src/select.coffee]: I used git revert, I think this is from before.

Comment (thangdinh) [inline src/select.coffee]: Your revert is right. This place is wrong not because of the revert. I fixed the position that's why it was in my commit.

**Pull request merged**